examples/kinetics_1.py

```diff
@@ -0,0 +1,170 @@
+#! /usr/bin/env python
+
+import matplotlib.pyplot as plt
+import scipy as sp
+import numpy as np
+import radicalpy as rp
+
+
+def main():
+    # Kinetic simulation of FAD at pH 2.3.
+    # For FAD quenching: uncomment the three quenching kinetic parameters.
+
+    # FAD kinetic parameters
+    kex = 1e4  # groundstate excitation rate
+    kfl = 3.55e8  # fluorescence rate
+    kic = 1.28e9  # internal conversion rate
+    kisc = 3.64e8  # intersystem crossing rate
+    khfc = 8e7  # spin-state mixing rate
+    kd = 3e5  # protonated triplet to ground state
+    k1 = 7e6  # protonated triplet to RP
+    km1 = 2.7e9  # RP to protonated triplet
+    krt = 1e9  # triplet state relaxation rate
+    kbet = 1.3e7  # singlet recombination rate
+    kr = 1.7e6  # RP relaxation rate
+    pH = 2.3 # pH of the solution
+    Hp = 10 ** (-1 * pH)  # concentration of hydrogen ions
+
+    # Quenching kinetic parameters
+    kq = 0  # 1e9  # quenching rate
+    kp = 0  # 3.3e3  # free radical recombination
+    Q = 0  # 1e-3  # quencher concentration
+
+    # Rate equations
+    base = {}
+    base["S0"] = {
+        "S0": -kex,
+        "T+/-": kd,
+        "T0": kd,
+        "S": kbet,
+        "S*": kfl + kic,
+        "Quencher": kp,
+    }
+    base["S*"] = {
+        "S*": -(kfl + kic + 3 * kisc),
+        "S0": kex,
+    }
+    base["T*+/-"] = {
+        "T*+/-": -(kd + k1 + krt),
+        "T+/-": km1 * Hp,
+        "T*0": 2 * krt,
+        "S*": 2 * kisc,
+    }
+    base["T*0"] = {
+        "T*0": -(kd + k1 + 2 * krt),
+        "T0": km1 * Hp,
+        "T*+/-": krt,
+        "S*": kisc,
+    }
+    base["Quencher"] = {
+        "Quencher": -kp,
+        "S": kq * Q,
+        "T+/-": kq * Q,
+        "T0": kq * Q,
+    }
+
+    off = {}
+    off["S"] = {
+        "S": -(3 * khfc + kbet),
+        "T+/-": khfc,
+        "T0": khfc,
+    }
+    off["T+/-"] = {
+        "T+/-": -(2 * khfc + km1 * Hp),
+        "S": 2 * khfc,
+        "T0": 2 * khfc,
+        "T*+/-": k1,
+    }
+    off["T0"] = {
+        "T0": -(3 * khfc + km1 * Hp),
+        "S": khfc,
+        "T+/-": khfc,
+        "T*0": k1,
+    }
+
+    on = {}
+    on["S"] = {
+        "S": -(2 * kr + khfc + kbet),
+        "T+/-": kr,
+        "T0": khfc,
+    }
+    on["T+/-"] = {
+        "T+/-": -(2 * kr + km1 * Hp),
+        "S": 2 * kr,
+        "T0": 2 * kr,
+        "T*+/-": k1,
+    }
+    on["T0"] = {
+        "T0": -(2 * kr + khfc + km1 * Hp),
+        "S": khfc,
+        "T+/-": kr,
+        "T*0": k1,
+    }
+
+    my_states = ["S0", "S*", "T*+/-", "T*0", "S", "T+/-", "T0", "Quencher"]
+    initial = [0, 0, 0, 0, 0, 2 / 3, 1 / 3, 0]
+    time = np.linspace(0, 6e-6, 200)
+
+    rates_off = {**base, **off}
+    rates_on = {**base, **on}
+    result_off = rp.classical.kinetics(time, initial, my_states, rates_off)
+    result_on = rp.classical.kinetics(time, initial, my_states, rates_on)
+    fac = 0.07
+
+    triplet_off = result_off[:, 2] + result_off[:, 3]
```

Comment:
Better grouping.

Reply:
Instead of:
```py
triplet_off = result_off[:, 2] + result_off[:, 3]
```
we'd like something like:
```py
triplet_off = select(result_off, ["T*+/-", "T*0"])
```